### ISSUE TITLE
fix / improve multi tree backups

### ISSUE THREAD

**Issue by willwade** [AceCentre/pasco#283]
Right now having a vocabulary setup with multiple trees - and different locales won't work in pasco. We could do with improving that

**Comment by gavinhenderson** [comment]
@willwade Could you clarify this slightly for me, I have a few questions. 

I had a play around and looked into the `<meta data-locale="en">` tag. Does this not solve the problem? If you have a 'german tree' then you can tag every root node with `<meta data-locale="de">` and then in the 'english tree' you can do `<meta data-locale="en">` for each root node?

Then when you switch boards with the `<meta data-change-tree-by-name="german-tree">` it will switch locale as you go between boards? 

I had a quick try and from what I can see it seems to work? It reads the 'german tree' in a german voice and switches back to english when i change back boards.

I feel like im missing something, is changing locale meant to do more than just change the voice?

**Comment by willwade** [comment]
Yep. I think I wrote this before realising this.  But I think the real issue Is that when you export its only exporting the one tree currently active. Not any of the others that that tree is calling or linked to (inactive trees). That make sense? So a feature of export / import really

**Comment by gavinhenderson** [comment]
@willwade Yeah I get you. I just checked out the current import/export and it looks like we can export linked boards and import them and they stay linked. You do have to do the import and export for each board rather than it doing them all but if you do them one at a time it does preserve the link.

Maybe we need a 'full export' setting which dumps all the boards and a 'full import' which allows you to insert all the boards? 

It sounds like a nice feature but is it one we really need? Im tempted to close this issue for now and wait to see if it comes up again?

**Comment by willwade** [comment]
Granted - its a real PITA when setting anyone up with multilingual trees. Particularly if they are also using prediction libraries. Its hideous. I'd vote for doing it if poss!

**Comment by gavinhenderson** [comment]
Gotcha, ill make a new issue to do the bulk import / export!
